day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,44 @@
+10.downto(1).each do |x|
+
+  new_frog_number = x - 1
+  if new_frog_number > 1 || new_frog_number == 0
+    was_were = "were"
+  else
+    was_were = "was"
+  end
+
+  if new_frog_number > 1 || new_frog_number == 0
+    new_frog_plural = "frogs"
+  else
+    new_frog_plural = "frog"
+  end
+
+  if x > 1
+    frog_plural = "frogs"
+  else
+    frog_plural = "frog"
+  end
+
+  if x > 1
+    jumper = "One"
+  else
+    jumper = "It"
+  end
+
+  puts "#{x} speckled #{frog_plural} sat on a log
+  eating some most delicious bugs.
+  #{jumper} jumped in the pool where it's nice and cool,
+  then there #{was_were} #{new_frog_number} speckled #{new_frog_plural}."
+
+end
+
```

Comment:
Nice work just having 1 area where you type the whole poem out! It can be easy here to go down a path where you type this out in multiple locations in your code.